salt/etcd/create-or-update-etcd-pillar.jinja

```diff
@@ -0,0 +1,81 @@
+#!/bin/env python
+
+import json
+import subprocess
+
+
+# This is set by salt, with jinja, when copying the file over to the minion
+CRI = '{{ salt.caasp_cri.cri_name() }}'
+
+
+def _cri_runtime_endpoint():
+    '''
+    Return the path to the socket required by crictl to communicate
+    with the CRI
+    '''
+    # check the crio socket first, there might be cases when docker and crio
+    # are running at the same time
+    if CRI == 'docker':
+        return '/var/run/dockershim.sock'
+    elif CRI == 'crio':
+        return '/var/run/crio/crio.sock'
+    else:
+        raise RuntimeError('Cannot find a suitable CRI socket')
```

Comment:
This could be simplified:

```Python
def _cri_runtime_endpoint():
    return {{ salt.caasp_cri.cri_runtime_endpoint }}
```

and, as this would be a constant,

```Python
_CRI_RUNTIME_ENDPOINT = {{ salt.caasp_cri.cri_runtime_endpoint }}
```

and you could remove the previous `CRI` variable...